nextflow/nextflow_examples.ipynb

```diff
@@ -0,0 +1,1108 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d872c44-451c-4e76-87b2-3ed6267822b5",
+   "metadata": {},
+   "source": [
+    "**If your workspace is NOT a clone of the [Workflows Getting Started workspace](http://todo),** you'll need to run the cell below to clone the GitHub repository containing nf-core test data into this cloud environment, then check out the branch containing RNASeq-specific test datasets and sample sheets. Run the following cell to validate that the referenced resource exists and check out the appropriate branch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5aceddab-e1c0-411d-8f50-07aca1b4d38a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "![[ -f test-datasets/samplesheet/samplesheet_minimal.csv ]] && echo \"Resource already exists\" || (terra resource add-ref git-repo --name=nf-core-sample-data-repo --repo-url=git@github.com:nf-core/test-datasets.git && cd /home/jupyter && terra git clone --resource=nf-core-sample-data-repo &&  git checkout rnaseq)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32a79377-dc63-49e0-a2d1-d03af407dc3e",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Run the RNASeq pipeline from `nf-core`\n",
+    "\n",
+    "\n",
+    "Run the cell below to launch the nf-core RNASeq pipeline with Google Cloud Life Sciences API as the executor on a dataset of reads from seven [_S. cerevisiae_](https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_000146045.2/) samples. Each run should take about one hour to complete. \n",
+    "\n",
+    "The nf_core RNASeq pipeline produces a large volume of informative output.\n",
+    "When inspecting the output, take note each time the executor you configured, the Google Life Sciences API, is invoked to execute the workflow steps that comprise RNASeq, from initial processing to alignment and creation of BAM files to creation of the MultiQC output files.\n",
+    "\n",
+    "When the pipeline run has completed successfully, the last few lines of output should resemble:\n",
+    "```\n",
+    "Waiting for file transfers to complete (1 files)\n",
+    "-[nf-core/rnaseq] Pipeline completed successfully-\n",
+    "Completed at: DD-Month-YYY HH:SS:MM\n",
+    "Duration    : 58m 56s\n",
+    "CPU hours   : <HOURS>\n",
+    "Succeeded   : <NUBMER of TASKS>\n",
+    "\n",
+    "Restoring the original gcloud project configuration: <GOOGLE-CLOUD-PROJECT-ID>\n",
+    "Updated property [core/project].\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e266fbc3-f018-42a4-9b18-9c3dec3a1f56",
+   "metadata": {
+    "tags": [
+     "\"output_scroll\"",
+     "\"scroll\":\"true\""
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "!terra nextflow -c /home/jupyter/nextflow.config run nf-core/rnaseq -profile google -w $(terra resource resolve --name=ws_files_autodelete_after_two_weeks)/nf_core/wd -r 3.11.1 \\\n",
```

Comment:
It's actually just a warning, not an error, and it cannot be disabled. Since there's a canonical config file for all `google` in `nf-core`, this is simply alerting the user that they're overriding the defaults here. (Which we definitely want them to do, for example -- we don't want the default zone, `europe-west2-c`!)

Reply:
I figured out a way to address with a different version of the config file that AFAICT is working properly-- let's talk more on Monday and I'll show you what I did.

Reply:
fixed!